app/services/retrieve_position_data.rb

```diff
@@ -47,10 +47,15 @@ def query_format
         OPTIONAL { ?position pq:P768 ?district . }
         OPTIONAL { ?position pqv:P580 [wikibase:timeValue ?position_start; wikibase:timePrecision ?position_start_precision] . }
         OPTIONAL { ?merged_then_deleted owl:sameAs ?person }
+        BIND(ceil((year(?term_start) - year(?position_start)) * 365.2425 +
+                  (month(?term_start) - month(?position_start)) * 30.4375 +
+                  (day(?term_start) - day(?position_start))) AS ?days_between)
```

Comment:
`?days_between` as a name leaves me uncertain about which direction the number will be postive in, if you see what I mean. Perhaps `?days_before_term_start` would make it easier to understand the usage later?

Reply:
Changed